deploy/CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ if(WIN32)
     if(MINGW)
         set(_ZLIB_DLL_FILENAME "libzlib.dll")
     else()
-        set(_ZLIB_DLL_FILENAME "zlib.dll")
```

Comment:
This will cause the windows CI to break the dll built on the windows CI is named "zlib.dll". May need to use a regex for the something like `(lib)?zlib(1)?.dll` . I've been thinking about changing this also in ff7tk (as zlib is needed for ff7tkutils )

Reply:
No more required. But thanks!